docs/operations/memory_hotplug.md

```diff
@@ -0,0 +1,136 @@
+# Memory Hotplug
+
+Memory hotplug was introduced in KubeVirt version 1.1, enabling the dynamic resizing of the amount of memory available to a running VM.
+
+# Configuration
+
+### Enable feature-gate
+
+To use memory hotplug we need to add the `VMLiveUpdateFeatures` feature gate in the Kubevirt CR:
+
+```yaml
+apiVersion: kubevirt.io/v1
+kind: KubeVirt
+spec:
+  configuration:
+    developerConfiguration:
+      featureGates:
+        - VMLiveUpdateFeatures
+```
+
+### Configure the Workload Update Strategy
+
+Configure `LiveMigrate` as `workloadUpdateStrategy` since the current implementation of the hotplug process requires the VM to live-migrate.
```

Comment:
This might seem obvious but could we also mention that this action is performed in the KubeVirt CR as well